apps/runtime/core/CONTRIBUTING.md

```
@@ -0,0 +1,52 @@
+docker build -f apps/runtime/core/Dockerfile --target dev -t celerity-runtime-core:dev .
+```
+
+## What belongs here
+
+Only the process around the runtime including reading the environment, starting the runtime, starting and supervising the handlers executable, and shutdown ordering. Everything the runtime itself does belongs in `libs/runtime/core` under the `lib-rt-core` commit scope, where it is shared with the language-specific runtimes.
+
+## Commit scope
+
+`runtime-core`
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)